docs/user/guides/network.md

```diff
@@ -24,55 +20,58 @@ sudo cp /mnt/LiveOS/squashfs.img /srv/tftpboot/boot/images/agama/x86_64/
 
 How to set up a PXE server is out of scope of this guide but some documentation for SLES can be found at [PXE Server](#pxe-server-documentation).
 
-## Important changes compared YaST
+## Important changes from YaST installer
 
 There are some important changes that affect the installation workflow and specially the [kernel boot options](#kernel-boot-options).
 
-- Linuxrc is not included and dracut takes care of booting the installer.
+- Linuxrc is not included and [dracut](https://man7.org/linux/man-pages/man7/dracut.cmdline.7.html) takes care of booting the installer.
```

Comment:
You do not need to repeat the link everytime you mention dracut. But, it would be much easier if you do something like:

```
[the text you want][dracut]

[dracut]: https://man7.org/linux/man-pages/man7/dracut.cmdline.7.html
```